### PR TITLE
magento/devdocs#: Add info about setPaymentMethodAndPlaceOrder to GraphQL checkout tutorial

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about `setPaymentMethodAndPlaceOrder` mutation and provides additional changes for "Step 4. Set the shipping address", "Step 5. Set billing address" "Step 6. Set the shipping method" pages.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.html
- https://devdocs.magento.com/guides/v2.3/graphql/tutorials/checkout/checkout-coupon.html
- https://devdocs.magento.com/guides/v2.3/graphql/tutorials/checkout/checkout-quote-email.html 

## Links to Magento source code

- [setPaymentMethodAndPlaceOrder](https://github.com/magento/graphql-ce/pull/723)

Thank you!

CC: @keharper

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md [hunk 1]
```diff
@@ -96,6 +106,44 @@ If the operation is successful, the response contains the code of the selected p
 }
 ```
 
+### Set payment method and place order {#setPaymentMethodAndPlaceOrder}
+
+Use the `setPaymentMethodAndPlaceOrder` mutation to set the payment method for your order.
```
Comment (rogyar) [inline guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: Please, add a clarification here 

```suggestion
Use the `setPaymentMethodAndPlaceOrder` mutation to set the payment method and place the order.
```

Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: good point @rogyar 

**fixed**

File: guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md [hunk 2]
```diff
@@ -96,6 +106,44 @@ If the operation is successful, the response contains the code of the selected p
 }
 ```
 
+### Set payment method and place order {#setPaymentMethodAndPlaceOrder}
```
Comment (rogyar) [inline guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: I've just realized that this mutation is not a part of 2.3.2 version. I would suggest splitting this PR into two separate PRs and making the PR with information about `setPaymentMethodAndPlaceOrder` mutation to the develop branch.

Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: I've created a separate PR https://github.com/magento/devdocs/pull/5000 to fix 
1. Remove `address_type` from `shipping_addresses` and `billing_addresses`.
2. Remove `label` from `selected_shipping_method`

Comment (rogyar) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: Thank you. The only change left is to create a PR to the `develop` branch.

Comment (keharper) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: @rogyar I will reset the target branch for this PR into the `develop` branch immediately after the next time we sync `master` and `develop`.

File: guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md [hunk 3]
```diff
@@ -96,6 +106,44 @@ If the operation is successful, the response contains the code of the selected p
...
+    order {
+      order_id
+    }
+  }
+}
+```
+
+**Response**
+
+If the operation is successful, the response contains the code of the selected payment method.
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: order ID, @keharper

[the remark this answers is not in the document]
Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: **fixed**

**Comment by keharper** [comment]
running tests

File: guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md [hunk 4]
```diff
@@ -54,6 +54,16 @@ query {
 }
 ```
 
+There are two mutation queries in GraphQl which can be use to set the payment method for your order:
+
+|Mutation|Description|
+|--- |--- |
+|`setPaymentMethodOnCart`|Sets the payment method for your order|
+|`setPaymentMethodAndPlaceOrder`|Sets the payment method and then immediately places your order. In this case ["Step 10. Place the order"]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-place-order.md) can be skipped|
```
Comment (keharper) [inline guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]:
```suggestion
|`setPaymentMethodAndPlaceOrder`|Sets the payment method and then immediately places your order. In this case ["Step 10. Place the order"]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-place-order.html) can be skipped|
```

Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: Thank you, @keharper

Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: fixed by @keharper ;)

Comment (keharper) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: @atwixfirster I can't commit the change. I can only make a suggestion to fix it

Comment (atwixfirster) [reply guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md]: Not sure how but you fixed that bug. I've pulled the changes and detected what `md` extension has been replaced with `html`.

Please run tests now.

Thank you!

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @atwixfirster

**Comment by meker12** [comment]
@keharper Please add the whatsnew for this PR. Thx.

**Pull request merged**